projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -322,7 +322,7 @@ private function load_value_from_callback( $item, $callback, $default = '' ) {
 
 	/**
 	 * Loads any extra data for a task, calling the 'extra_data_callback' callback if it exists,
-	 * or an empty array if there is no callback.
+	 * or an null if there is no callback or the callback returns an empty array.
```

Comment:
Nit: we can drop the `an` here.
```suggestion
	 * or null if there is no callback or the callback returns an empty array.
```